satpy/etc/enhancements/generic.yaml

```diff
@@ -79,6 +79,22 @@ enhancements:
     - name: gamma
       method: *gammafun
       kwargs: {gamma: 1.8}
+  fire_temperature_default:
```

Comment:
Is this enhancement specific to VIIRS? Maybe we should put it in `enhancements/viirs.yaml`?

Reply:
I guess the same enhancement can be applied to any sensor with a 1.6, 2.2 and 3.7 micron channel....

Reply:
I will continue on Tuesday, possibly tuning it further. But now Easter holiday. So, @djhoese 
don't merge yet. Just wanted to add it before closing down for today.

Reply:
![capture_viirs_201507042209_alaska_fire_temperature_298368_20100_1595_crop](https://user-images.githubusercontent.com/169069/56620292-b56c2800-6628-11e9-9472-e1423fde80fa.jpg)
And here the one from http://rammb.cira.colostate.edu/projects/alaska/blog/index.php/tag/fire-temperature-rgb/
![fire_temperature_20150704_2209Z](https://user-images.githubusercontent.com/169069/56620297-b7ce8200-6628-11e9-8bb2-6800a14e7b4d.jpg)

Reply:
After some further tuning...

Reply:
![capture_viirs_201507042209_alaska_fire_temperature_298368_20130_15130_gamma_11_12_12_crop](https://user-images.githubusercontent.com/169069/56655746-fbaaa100-6693-11e9-8fd3-57cd06aafef2.jpg)

...after a bit of further tuning with @mraspaud using a bit of gamma we are even closer. This is my final bet!